_shows/19_20/julius_caesar.md

```diff
@@ -0,0 +1,62 @@
+---
+
+title: "[The Fall of] Julius Caesar"
+playwright: William Shakespeare
```

Comment:
Presumably this was adapted by someone too?

Reply:
that information is not on the spreadsheet

Reply:
Added